kinotab_back/profiles/signals.py

```diff
@@ -0,0 +1,16 @@
+from django.contrib.auth.models import User
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from .models import Profile
+
+
+@receiver(post_save, sender=User)
+def create_user_profile(sender, instance, created, **kwargs):
```

Comment:
It's seems thast kwargs not used evereywere. Use **_ instead **kwargs
```suggestion
def create_user_profile(sender, instance, created, **_):
```